### PR TITLE
Approved revs

### PR DESCRIPTION
**Pull request by krisfield**
Integrates [Extension:Approved Revs](https://www.mediawiki.org/wiki/Extension:Approved_Revs) version 1.0 or later into Special:Pending Reviews.

### CODE REVIEW

File: specials/SpecialPendingReviews.php [hunk 1]
```diff
@@ -125,27 +125,58 @@ public function execute( $parser = null ) {
...
-			if ( $item->title ) {
+			if ( !( $useApprovedRevs && is_a( $item, 'PendingApproval' ) && $item->title ) ) {
 				$html .= $this->getStandardChangeRow( $item, $rowCount );
+			}
 
 			// page has been deleted (or moved w/o a redirect)
-			} else {
-				$html .= $this->getDeletedPageRow( $item, $rowCount );
-			}
+ elseif ( !( $useApprovedRevs && is_a( $item, 'PendingApproval' ) ) ) {
```
Comment (krisfield) [inline specials/SpecialPendingReviews.php]: fix indent

File: specials/SpecialPendingReviews.php [hunk 2]
```diff
@@ -125,27 +125,58 @@ public function execute( $parser = null ) {
...
+					$html .= $this->getDeletedPageRow( $item, $rowCount );
```
Comment (krisfield) [inline specials/SpecialPendingReviews.php]: fix indent

File: specials/SpecialPendingReviews.php [hunk 3]
```diff
@@ -125,27 +125,58 @@ public function execute( $parser = null ) {
...
+	}
 
-			$rowCount++;
+			 $rowCount++;
```
Comment (jamesmontalvo3) [inline specials/SpecialPendingReviews.php]: space in front

**Comment by jamesmontalvo3** [comment]
Okay...so looks like all the nit-picking is fixed. As for the actual function...is it complete in your opinion?

**Comment by jamesmontalvo3** [comment]
How about now? I'd like to know if it's going to have any more changes before relooking through code again. Function on the dev server you added it to looks good.

**Comment by krisfield** [comment]
Yes, i’m all done with changes.

**Comment by krisfield** [comment]
The changes I made fix the `typeError` bug we were seeing for Special:PendingReview pages that had "deleted page" notifications. 

I am done making edits (again)

**Pull request merged**